index.js

```diff
@@ -75,7 +85,9 @@ app.on('ready', () => {
 
     rpc.on('init', () => {
       win.show();
-      if (fn) fn(win);
+      if (!fn) fn = (win) => win.rpc.emit('session add req');
+      (app.windowCallback || fn)(win);
```

Comment:
I realize this is way late, but what does `open file` do that doesn't want `termgroup add req` (previously `session add req`) to be fired?

Any other plugin authors that want to rely on `createWindow` to open a "regular window," needs to know they need to call `win.rpc.emit('termgroup add req')`--which isn't intuitive--otherwise their window appears "broken" (it looks like an empty window). I haven't used Hyper in a while, but it appears that this broke a few things because of the new requirement of calling `win.rpc.emit('termgroup add req')` (which has already changed once).

Hopefully there is some way that we can address both use-cases more cleanly.